default.yaml

```diff
@@ -816,6 +805,23 @@ roleplay:
     # add the real nickname, in parentheses, to the end of every roleplay message?
     add-suffix: true
 
+# external services can integrate with the ircd using JSON Web Tokens (https://jwt.io).
+# in effect, the server can sign a token attesting that the client is present on
+# the server, is a member of a particular channel, etc.
+extjwt:
+    # default service:
+    # expiration: 45s
+    # symmetric secret for HMAC signing:
+    # secret: "65PHvk0K1_sM-raTsCEhatVkER_QD8a0zVV8gG2EWcI"
+    # private key for RSA signing:
+    # rsa-private-key-file: "extjwt.pem"
```

Comment:
this line is different between default and conventional

this config section feels a bit clunky because it's hard to tell what's a key and what's not a key, e.g. is `default service:` a dict key that you should fill a value into, etc.

Something like this feels like it should work?
```yaml
extjwt:
    # # default service:
    # expiration: 45s
    # # symmetric secret for HMAC signing:
    # secret: "65PHvk0K1_sM-raTsCEhatVkER_QD8a0zVV8gG2EWcI"
    # # private key for RSA signing:
    # rsa-private-key-file: "extjwt.pem"

    # # named services:
    # services:
    #     "jitsi":
    #         expiration: 30s
    #         secret: "qmamLKDuOzIzlO8XqsGGewei_At11lewh6jtKfSTbkg"
```